### DISCUSSION TITLE
Chained queries (mixed datasources) in the same panel

### DISCUSSION THREAD

**Discussion by olivierperdaens** [grafana/grafana#47755] [Dashboards, Panels & Visualizations]
Hi,

Working with the latest version of Grafana (8.4.6), I’m trying to use the result of a query to build a second query (from a different Datasource) on the same panel. I can’t manage to make this work. Is this functionality available ?

For the context, I have a timeseries database with my measurements (let’s say CO2 measurement) and somes location tag (eg. room and building number). I would like to query the measurements, and use the location tag to query on another database to retrieve more information about the location (eg. room name, room surface, room occupancy, …)

Thanks for the help :wink:

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
